gem/styles/gem-springster/state/state.scss

```diff
@@ -17,6 +17,9 @@
       margin-left: 10px;
     }
   }
+  #footer-bar .footer-logo {
+    width: 200px;
```

Comment:
It actually needs a width otherwise it doesn't show the image.
I need to deploy to test this unfortunately